py3-pystache.yaml

```
@@ -0,0 +1,80 @@
+package:
+  name: py3-pystache
+  version: "0.6.8"
+  epoch: 0
+  description: "A mustache template renderer for Python"
+  copyright:
+    - license: MIT
+
+vars:
+  pypi-package: pystache
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+      3.13: "313"
+
+environment:
+  contents:
+    packages:
+      - py3-supported-build-base-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/PennyDreadfulMTG/pystache.git
+      tag: v${{package.version}}
+      expected-commit: 531a0ecea00a696423175ef46c319b29408dc3e2
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provides:
+        - py3-${{vars.pypi-package}}
+      provider-priority: ${{range.value}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
+    test:
+      environment:
+        contents:
+          packages:
+            - python-${{range.key}}
+      pipeline:
+        - runs: |
+            pystache -h | grep "Usage"
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            import: ${{vars.pypi-package}}
+        - runs: |
+            # Test the package
+            cat <<EOF > test.py
+            import pystache
+            print(pystache.render('Hi {{person}}!', {'person': 'World'}))
+            EOF
+            python test.py | grep -q 'Hi World!'
+
```

```suggestion
  - range: py-versions
    name: py${{range.key}}-${{vars.pypi-package}}-bin
    description: Executable binaries for ${{vars.pypi-package}} installed for python${{range.key}}
    dependencies:
      runtime:
        - py${{range.key}}-${{vars.pypi-package}}
    pipeline:
      - runs: |
          mkdir -p ${{targets.contextdir}}/usr
          mv ${{targets.contextdir}}/../py${{range.key}}-${{vars.pypi-package}}/usr/bin ${{targets.contextdir}}/usr
    test:
      pipeline:
        - runs: |

```

You do need a bin package so these packages don't overlap.  Something like this should work where the runs is doing `--help` or `--version` for the commands that are being added.